CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,7 @@
 # Code of Conduct
 
 Be respectful toward other members of the community. Don't initiate or engage in discussions that are designed to insult, inflame, attack, or incite hate/discrimination/bullying against others. Don't feed the trolls. If we detect any kind of behaviour we consider unacceptable, we reserve the right to ban, block, or remove the involved members from any or all of our communities.
+
+# See Also
+
+- [Rules for managing Pull Requests](PULL_REQUEST_MANAGEMENT.md)
```

Comment:
This doesn't belong to CoC.

Reply:
To give that document some exposure, it was decided in the last meeting to reference it from the Code of Conduct as well.
Plus, it contains some guidelines "in order to avoid coming off as rude to helpful contributors", so I don't think it's wrong to link it from here.